src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <>
+      <section className="dp-gray-page p-t-54 p-b-54">
+        <div className="dp-container">
+          <div className="dp-rowflex">
+            <div className="col-lg-6">
+              <div>
+                <BigQueryLogo></BigQueryLogo>
+                <h1>{_('big_query.free_title', '')}</h1>
+              </div>
+              <p>{_('big_query.free_text_summary', '')}</p>
+              <div>
+                <ul className="dp-feature-list">
+                  <li>
+                    <span className="dp-icodot">.</span>
+                    <span>{_('big_query.free_ul_item_one', '')}</span>
+                  </li>
+                  <li>
+                    <span className="dp-icodot">.</span>
+                    <span>{_('big_query.free_ul_item_two', '')}</span>
+                  </li>
+                </ul>
+              </div>
+              <p>
+                <strong>{_('big_query.free_text_strong', '')}</strong>
+              </p>
+              <hr />
+              <div>
+                <a
+                  href={_('big_query.upgrade_plan_url', '')}
+                  className="dp-button button-medium primary-green"
+                >
+                  {_('big_query.free_btn_redirect', '')}
+                </a>
+              </div>
+            </div>
+            <div className="col-lg-6">
+              <img src={datastudiogif} width="800" height="500" />
+              <p className="text-italic">
+                {_('big_query.free_text_data_studio_one', '')}{' '}
+                <a href={_('big_query.free_url_data_studio', '')}>
+                  {_('big_query.free_text_data_studio_two', '')}
+                </a>
+                {_('big_query.free_text_data_studio_three', '')}{' '}
```

Comment:
It would be nice having all these items in the same resource.

@fcoronelmakingsense you are the expert on this, could you help @kapoc3 on this?

Reply:
`<p className="text-italic">
{_('big_query.free_text_data_studio, '')}{' '}
</p>`
and i put the other change in the language file

Reply:
<pre>
<FormattedMessageMarkdown
                linkTarget={'_blank'}
                id="big_query.free_text_data_studio_MD"
              />
</pre>

thats better